core/src/main/scala/cats/SelectiveError.scala

```diff
@@ -0,0 +1,15 @@
+package cats
+
+import cats.syntax.either._
```

Comment:
This is for a handle on our cached `Left(())`.  I think syntax in the typeclasses is frowned on.